instapy_cli/media.py

```diff
@@ -1,19 +1,23 @@
 import os, requests
+from PIL import Image
 #import urlparse for Python2 and Python3
+
 try:
     from urllib.parse import urlparse
 except ImportError:
     from urlparse import urlparse
 
+MIN_ASPECT_RATIO = 0.80
+MAX_ASPECT_RATIO = 1.91
 
 class Media(object):
     pathFile = None
+    optimizedImage = None
```

Comment:
Did not want to override the original object. So added this. Let me know if this is fine!